src/Model/Group.php

```diff
@@ -13,15 +13,13 @@
 
 class Group
 {
-    private $identifier;
-    private $name;
     /** @var string[] */
-    private $userIds = [];
+    private array $userIds = [];
 
-    public function __construct(string $identifier, string $name)
-    {
-        $this->identifier = $identifier;
-        $this->name = $name;
+    public function __construct(
+        private string $identifier,
+        private string $name
+    ) {
```

Comment:
Same question for such cases

Reply:
For the other class, I didn't make it that way so I don't know why it is like that.
But for this case, I believe I just changed the way of instantiating the construct parameters to the new PHP8.0 way, or did I fail somewhere ?